oasis-std/src/backend/wasi.rs

```diff
@@ -43,6 +43,10 @@ pub fn payer() -> Address {
     env_addr("PAYER")
 }
 
+pub fn aad() -> Vec<u8> {
+    hex::decode(&std::env::var_os("AAD").unwrap().into_vec()).unwrap()
```

Comment:
please, please anything but hex. base64, perhaps?

Reply:
OK :D
I'll fix the other PR too.

Reply:
Done! :)